src/controllers/commands/CommandController.cpp

```diff
@@ -3068,6 +3068,67 @@ void CommandController::initialize(Settings &, Paths &paths)
 
             return "";
         });
+
+    auto uniqueChatLambda = [formatChatSettingsError](auto words, auto channel,
+                                                      bool target) {
```

Comment:
There was enough identical logic that I chose to just pass in a `target` bool variable to allow you to toggle r9k modes instead.